resources/eds_templates/diff_report/templates/sections/added_reified_concept_notation_value.html

```diff
@@ -29,19 +29,12 @@ <h2 class="ui header">Added concept reified notations</h2>
 PREFIX sd: <http://www.w3.org/ns/sparql-service-description#>
 PREFIX sh: <http://purl.org/skos-history/>
 PREFIX xhv: <http://www.w3.org/1999/xhtml/vocab#>
-SELECT DISTINCT ?instance (str(?prefLabel) as ?instanceLabel) (?value AS ?addedValue)
+SELECT DISTINCT ?instance (str(?prefLabel) as ?instanceLabel) (str(?value) AS ?addedValue) ?languageTag
```

Comment:
no language tag in notations pls. Only in labels and notes (both reified and simple)